sass/pages/_assets.scss

```diff
@@ -90,22 +90,34 @@
             }
         }
 
+        &__bevy-versions {
+            grid-area: versions;
+        }
+
+        &__licenses {
+            grid-area: licenses;
+            margin-left: auto;
+            order: 2;
+        }
+
+        &__tag-list {
+            display: flex;
+            align-items: center;
+            gap: .2em;
+        }
+
         &__tag {
-            padding: .3em .5em;
-            margin-right: .5em;
-            border-radius: .3em;
+            display: flex;
+            justify-content: center;
+
+            padding: .2em .4em;
+            border-radius: .2em;
             font-size: .85em;
             background: rgb(89, 89, 94);
-            color: $color-white;
 
             &--supported {
                 background-color: rgb(110, 110, 247);
             }
         }
-
-        a {
-            text-decoration: none;
-            color: $link-color;
-        }
     }
 }
```

Comment:
Nit: Can we add a trailing linebreak here? GitHub complains about that

Reply:
Is it possible you didn't refresh? My latest commit should have already fixed that one